haystack/nodes/ranker/diversity.py

```diff
@@ -2,7 +2,7 @@
 from pathlib import Path
 from typing import List, Literal, Optional, Union
 
-from haystack.nodes import BaseRanker
+from haystack.nodes.ranker.base import BaseRanker
```

Comment:
Shouldn't be necessary, right? Does it cause problems to import from `haystack.nodes`?

Reply:
@ZanSara I get circular dependency with that kind of import. But I see that other existing rankers also import from `haystack.nodes.ranker.base`